src/ConfigFileParser.cpp

```diff
@@ -330,6 +347,37 @@ bool ConfigFileParser::readAndTrimLine(const std::string& content, char delimite
 	return true;
 }
 
+/**
+ * @brief Reads the path of a location block
+ *
+ * If the path is "/", the m_locationIndex will be set to 0 because the default location has an index of 0
+ * Therefore the following functions will store the values correctly in the default location
+ *
+ * Otherwise a new location will be created, added to the locations vector and the m_locationIndex will be incremented
+ *
+ */
+void ConfigFileParser::readLocationBlockPath(void)
+{
+	std::string directiveLocation = "location";
+	size_t startIndex = directiveLocation.length();
```

Comment:
This will always be 8. Just having 8 would be hard to read.
You could use sizeof() to not create intermediate string and eval at compile time: 
`const size_t startIndex = sizeof("location") - 1; //subtract zero terminator`
Or just a constant: `const size_t startIndex = 8; // Length of "location"`

Reply:
Good point. I also did not want to write just 8.

I think your first suggestion fits in this case good and I used it for the purpose of calculating the index value.